src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (email.equals("bob@i.ua") && password.equals("1234")) {
+            return true;
+        }
+        if (email.equals("alice@i.ua") && password.equals("1234")) {
+            return true;
+        }
```

Comment:
Why two same if-statements? 
```suggestion
        if (email.equals("bob@i.ua") && password.equals("1234")) {
            return true;
        }
        if (email.equals("alice@i.ua") && password.equals("1234")) {
            return true;
        }
```